lambdas/service/app.py

```diff
@@ -771,36 +806,76 @@ def get_order():
     return {'order': Plugin.load().service_config().order_config}
 
 
-@app.route('/manifest/files', methods=['GET'], cors=True)
+@app.route('/manifest/files', methods=['GET'], cors=True, spec={
+    'description': 'Initiate and check status of a manifest generation job, returning a either a 301 redirecting to a '
+                   'URL to re-check the status of the manifest generation or a 302 redirecting to the location of the '
+                   'completed manifest.',
+    'parameters': [
+        {
+            'in': 'query',
+            'name': 'filters',
+            'description': 'Filters to be applied when generating the manifest',
+            'required': False,
+            'schema': {
```

Comment:
I think, instead of "schema", you can specify 

```
"content": {
  "application/json": {
    "schema": {
      "$ref": "#/components/schemas/..."
    }
  }
}
```

And then just start some empty template in components/schemas. See https://swagger.io/docs/specification/describing-parameters/#schema-vs-content

The schema for manifest parameters will be the same as the schema for file any other filter param for the other endpoints (right??). I don't think you need to bother to create the whole schema though, I think it's fair to leave that for someone else father down the line.